src/app/accounts/sso.component.ts

```diff
@@ -31,4 +32,23 @@ export class SsoComponent extends BaseSsoComponent {
         this.redirectUri = window.location.origin + '/sso-connector.html';
         this.clientId = 'web';
     }
+
+    async ngOnInit() {
+        const queryParamsSub = this.route.queryParams.subscribe(async (qParams) => {
+            if (qParams.code != null && qParams.state != null) {
+                if (qParams.state.endsWith(':clientId=browser')) {
+                    window.postMessage({ type: "AUTH_RESULT", code: qParams.code, state: qParams.state }, "*");
+                }
+            }
+
+            if (qParams.state.endsWith(':clientId=browser')) {
+                this.redirectUri = qParams.redirectUri;
+                this.state = qParams.state;
+                this.clientId = qParams.clientId;
+                await this.storageService.save(ConstantsService.ssoStateKey, qParams.state);
+            }
```

Comment:
still a little confused why this is needed.

Reply:
BaseSsoComponent isn't holding onto a value that's needed. If this isn't there .. instead of continuing the flow and redirecting to the extension, it redirects to the Org ID enter screen and stops.

Let me see if I can find the exact values and see if there is a way to handle them.

Reply:
state will come back in your redirect params. you should not need to save it.

Reply:
Removed. Seems to work fine without this.